testing/fil_token_integration/tests/frc46_single_actor_tests.rs

```diff
@@ -0,0 +1,146 @@
+use frc42_dispatch::method_hash;
+use frc46_token::token::{state::TokenState, types::MintReturn};
+use fvm_integration_tests::{dummy::DummyExterns, tester::Account};
+use fvm_ipld_blockstore::MemoryBlockstore;
+use fvm_ipld_encoding::RawBytes;
+use fvm_shared::{econ::TokenAmount, receipt::Receipt};
+
+mod common;
+use common::{construct_tester, TestHelpers, TokenHelpers};
+use test_actor::{action, ActionParams, TestAction};
+
+const BASIC_TOKEN_ACTOR_WASM: &str =
+    "../../target/debug/wbuild/basic_token_actor/basic_token_actor.compact.wasm";
+const TEST_ACTOR_WASM: &str = "../../target/debug/wbuild/test_actor/test_actor.compact.wasm";
+
+/// This covers several simpler tests, which all involve a single receiving actor
+/// They're combined because these integration tests take a long time to build and run
```

Comment:
This is just commentary, but I'm really surprised if breaking this into multiple `#[test]` functions adds any observable build/runtime. What kind of crappy dev environment is this (says Rust n00b)?

Reply:
Oh, this is to do with the fvm integration test environment, compiling to WASM and stuff? Ok